Signum.Engine/Operations/OperationLogic.cs

```diff
@@ -105,9 +106,55 @@ public static void Start(SchemaBuilder sb)
             sb.Schema.SchemaCompleted += () => RegisterCurrentLogs(sb.Schema);
 
             ExceptionLogic.DeleteLogs += ExceptionLogic_DeleteLogs;
+
+            OperationsToken.GetTypeEligibleOperations = (entityType) => OperationsToken_GetTypeEligibleOperations(entityType);
+            OperationToken.IsAllowedExtension = (operationSymbol, entityType) => OperationToken_IsAllowedExtension(operationSymbol, entityType);
+            OperationToken.BuildExtension = (entityType, operationSymbol, parentExpression) => OperationToken_BuildExpression(entityType, operationSymbol, parentExpression);
         }
     }
 
+    private static IEnumerable<OperationSymbol> OperationsToken_GetTypeEligibleOperations(Type entityType)

```

Comment:
maybe rename to GetElegibleTypeOperations?

Reply:
Yes, quite right
changed.